policy_definitions/update_management/policy_definition_qby_deploy_updates.json

```diff
@@ -0,0 +1,412 @@
+                    }
+                  },
+                  "timeZone": {
+                    "defaultValue": "W. Europe Standard Time",
+                    "type": "String"
+                  },
+                  "startDate": {
+                    "defaultValue": "[utcNow('yyyy-MM-dd')]",
+                    "type": "string"
+                  },
+                  "vmName": {
+                    "type": "string",
+                    "metadata": {
+                      "description": "Name of the Virtual Machine."
+                    }
+                  },
+                  "vmResourceId": {
+                    "type": "string",
+                    "metadata": {
+                      "description": "ResourceId of the Virtual Machine."
+                    }
+                  },
+                  "patchModeSetting": {
+                    "type": "bool",
+                    "metadata": {
+                      "description": "patchModeSetting of the Virtual Machine."
+                    }
+                  },
+                  "patchMode": {
+                    "type": "string",
+                    "metadata": {
+                      "description": "patchMode of the Virtual Machine."
+                    }
+                  }
+                },
+                "variables": {
+                  "updateOptions": {
+                    "C": "Critical",
+                    "S": "Security",
+                    "U": "UpdateRollup",
+                    "F": "FeaturePack",
+                    "E": "ServicePack",
+                    "D": "Definition",
+                    "T": "Tools",
+                    "G": "Updates"
+                  },
+                  "rebootOptions": {
+                    "reboot": "IfRequired",
+                    "alwaysreboot": "Always",
+                    "neverreboot": "Never"
+                  },
+                  "splitSeverityGroup": "[split(parameters('severityGroupName'),'-')]",
+                  "recurEvery": "[concat(variables('splitSeverityGroup')[0], 'Month', ' ', variables('splitSeverityGroup')[1],' ', variables('splitSeverityGroup')[2])]",
+                  "startTime": "[concat(substring(variables('splitSeverityGroup')[3], 0, 2), ':', substring(variables('splitSeverityGroup')[3], 2, 2)  )]",
+                  "startDateTime": "[concat(parameters('startDate'), ' ', variables('startTime'))]",
+                  "updateClassification": "[map(range(0, sub(length(replace(variables('splitSeverityGroup')[4],'X', '')), 1 )), lambda('i', variables('updateOptions')[substring(replace(variables('splitSeverityGroup')[4],'X',''), lambdaVariables('i'),1 )] )) ]",
+                  "rebootSetting": "[variables('rebootOptions')[variables('splitSeverityGroup')[5]]]",
+                  "uniqueDeploymentName": "[concat('NestedDeploymentName-', uniqueString(deployment().name))]",
```

Comment:
I don't see any technical issues with either approach. The compliance should show correctly no matter if the deployment fails since the policy rule and existence condition will be fulfilled, right?

For me personally it would be logical that the nested deployment fails and the first deployment succeeds. If you view it from the perspective of the policy the job is done successfully when a deployment is attempted, assuming that one of them will succeed. Or other way around a failed nested deployment doesn't mean that the first deployment was unsuccessful in what it was trying to accomplish